src/core/geometry/bundle/Bundle.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <string>
+#include "core/util/Identification.h"
+#include "core/geometry/element/Element.h"
+
+namespace semba::geometry::bundle {
+	typedef geometry::ElemId ElemIdForBundle;
+	typedef MatId MatIdForBundle;
+
+	class Bundle {
+	public:
+		Bundle(
+			const std::string& name,
+			MatIdForBundle materialId,
+			MatIdForBundle initialConnectorId,
+			MatIdForBundle endConnectorId,
+			const std::vector<ElemIdForBundle>& elemIds);
+
+		std::string const& getName() const;
+		std::string getName();
```

Comment:
Why do we need two? The first warranties that name won't be modified. And the second is a use case in which the caller needs a copy. I would remove the second because if the caller needs a modifiable copy of the name, it can build a string for himself.

Reply:
Addressed in commit e0876b681447e7be6b8220111352cf1fac6c7c78.